packages/tests/unit-tests/test-cases/declarations/src/nested-inherited.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+    ComponentBindings,
+    Nested,
+    OneWay,
+    TwoWay,
+} from "@devextreme-generator/declarations";
+
+@ComponentBindings()
+export class FakeNested {
+  @OneWay() baseProp = 0;
+}
+
+@ComponentBindings()
+export class WidgetProps {
+  @OneWay() baseProp = 0;
+  @Nested() someProp: FakeNested = new FakeNested();
+}
+@ComponentBindings()
+export class TooltipProps {
+  @OneWay() tooltipValue = 0;
+  @Nested() tooltipNested: WidgetProps[] = [new WidgetProps()];
+}
+@ComponentBindings()
+export class BulletProps extends WidgetProps {
```

Comment:
Did you check the deeper inheritance? It seems that it may not works.

Reply:
I added deeper inheritance to test, fixed naming for angular, seems like working